modules/git/src/workflow.ts

```diff
@@ -173,13 +175,16 @@ export class StagingWorkflow {
 				step,
 			});
 
-			await run({
-				name: 'Hide unstaged changes',
-				cmd: 'git',
-				args: [...skipHooks, 'checkout', '--force', '--', ...files],
-				runDry: true,
-				step,
-			});
+			const files = processRenames(partiallyStaged, false);
+			if (files.length) {
```

Comment:
so if no files were renamed we dont hide the staged changes? or am i misinterpreting processRenames()

Reply:
`git checkout` is going to make any files that _aren't_ staged go back to their original state. It seems like with `git checkout --force --` (and no paths) that it's forcing staged files to get reverted.